src/core/img/icons/magnifying-glass.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="56px" height="56px" viewBox="-4 25 56 56" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
If this supercedes that you can problably delete the placeholder searchbox svg now...